scalus-core/shared/src/test/scala/scalus/cardano/plutus/contract/blueprint/BlueprintTest.scala

```diff
@@ -0,0 +1,39 @@
+package scalus.cardano.plutus.contract.blueprint
+import io.bullet.borer.Cbor
+import org.scalatest.funsuite.AnyFunSuite
+import scalus.builtin.{platform, ByteString, Data}
+import scalus.cardano.plutus.contract.blueprint.model.PlutusVersion.{v2, v3}
+import scalus.ledger.api.v3.{TxInfo, TxOutRef}
+import scalus.prelude.{Validator, *}
+import scalus.utils.Hex
+import scalus.cardano.plutus.contract.blueprint
+
+class BlueprintTest extends AnyFunSuite {
```

Comment:
Make a golden test: add an expected json to `resources` or just directly as a string constant in the test